runtime.ts

```diff
@@ -1127,7 +1127,7 @@ module J2ME {
         // Some Native MethodInfos are constructed but never called;
         // that's fine, unless we actually try to call them.
         return function missingImplementation() {
-          release || assert (false, "Method " + methodInfo.name + " is native but does not have an implementation.");
+          assert (false, "Method " + implKey + " is native but does not have an implementation.");
```

Comment:
Hmm, I'd rather stick with the convention that *assert* calls get compiled out in release builds (which is what `release || assert` does) and use *throw* here if we want to throw unconditionally. But the old VM merely logged via *console.error* in this case, so perhaps we should do the same.